ExtensionPack/package.json

```diff
@@ -0,0 +1,46 @@
+{
+    "name": "cpptools-extension-pack",
+    "displayName": "C/C++ Extension Pack",
+    "description": "Popular extensions for C++ development in Visual Studio Code.",
+    "publisher": "ms-vscode",
+    "icon": "LanguageCCPP_color_128x.png",
+    "readme": "README.md",
+    "author": {
+      "name": "Microsoft Corporation"
+    },
+    "license": "SEE LICENSE IN LICENSE.txt",
+    "version": "0.1.0",
+    "engines": {
+        "vscode": "^1.48.0"
```

Comment:
Why is 1.48 required?

Reply:
I have to investigate that, I don't think 1.48 is required. but I need to see what is the lowest vscode version for all.

Reply:
C/C++ "vscode": "^1.44.0"
CMAke "vscode": "^1.30.0"
GitHUb "vscode": "^1.48.0"

I guess we have to go with 1.48.0